### DISCUSSION TITLE
What's the best approach to delete specific row in in data-table ??

### DISCUSSION THREAD

**Discussion by Sahad-09** [shadcn-ui/ui#2512] [Q&A]
![image](https://github.com/shadcn-ui/ui/assets/91364349/ef68bdbd-9bb0-47b4-869c-311237dc429e)
![image](https://github.com/shadcn-ui/ui/assets/91364349/1beb96b0-b8c7-4152-9fc1-b4d30bedc9f1)
![image](https://github.com/shadcn-ui/ui/assets/91364349/496beeda-54fd-4209-bb6d-52d10b3a4465)

**Comment by ezeparziale** [comment]
Instead of using a Button, I utilize a DialogTrigger to display a confirmation Dialog. When the user clicks on the confirmation button, the server action or fetch is executed.

https://ui.shadcn.com/docs/components/alert-dialog

**Comment by victormicco-anjun** [comment]
You can use any component of shadcn to display it, over there has a lot of good and pretty components, I prefer to use Dialog or a AlertDialog, but you can use a Hover Card, or anything that will display it to you
https://ui.shadcn.com/docs/components/
